node/assembler/batch_fetcher.go

```diff
@@ -275,7 +275,7 @@ func (br *BatchFetcher) pullSingleBatch(ctx context.Context, batcherToPullFrom c
 		br.clientConfig,
 	)
 	if err != nil {
-		br.logger.Errorf("Assembler failed to pull batch %s from %v", types.BatchIDToString(batchID), batcherToPullFrom)
+		br.logger.Errorf("Assembler failed to pull batch %s from batcher %d with endpoint %s", types.BatchIDToString(batchID), batcherToPullFrom.PartyID, batcherToPullFrom.Endpoint)
```

Comment:
"... from batcher: Shard: %d, Party: %d, endpoint: %s",...

i.e. include the shard when specifying a batcher